sds_data_manager/stacks/api_gateway_stack.py

```diff
@@ -1,5 +1,7 @@
-from aws_cdk import Stack
```

Comment:
I notice that a fair number of the modules in this repo don't have module docstrings. I try to make it a habit to include these, but perhaps I am in the minority. Do we think it is important to include module docstrings?

Reply:
I agree, this repo is in a pretty inconsistent state right now :-/ I am +0.5 on module docstrings. So I added them in. I think in most of these cases it actually made sense to move the class docstrings up to the module level, so that is what I did.